src/components/sponsors/style.module.less

```diff
@@ -34,10 +34,46 @@
 
 	a {
 		text-decoration: none;
-		display: block;
+		display: inline-block;
```

Comment:
Before (notice how my house is under the Ayden logo but the upload.io logo is hovered):
<img width="812" alt="Screenshot 2023-01-12 at 2 24 40 PM" src="https://user-images.githubusercontent.com/459878/212194227-86248f58-84cf-4cd0-b527-aee3bc6ef376.png">

After (not nothing is highlighted when hovering whitespace):
<img width="569" alt="image" src="https://user-images.githubusercontent.com/459878/212195385-5598cc5f-8ef4-4305-bdfe-05b8f9469530.png">

After 2 (hovering over image directly highlights it):
<img width="561" alt="image" src="https://user-images.githubusercontent.com/459878/212195700-691ff255-6950-4795-9084-e72637efac6b.png">